example/simple/BUILD.bazel

```diff
@@ -43,3 +43,20 @@ jest_test(
         "index.test.js",
     ],
 )
+
+jest_test(
```

Comment:
This actually passed already because it's only the `*_update_snapshots` target that has the bug. Seems worth keeping though, just to test a generated config and that generation target is testonly.